src/robot/libraries/Process.py

```diff
@@ -407,7 +407,8 @@ def wait_for_process(self, handle=None, timeout=None, on_timeout='continue'):
         given in
         [http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#time-format|
         various time formats] supported by Robot Framework, for example, ``42``,
-        ``42 s``, or ``1 minute 30 seconds``.
+        ``42 s``, or ``1 minute 30 seconds``. ``${None}``, falsy,
+        zero, and negative values is same as not specifying timeout at all.
```

Comment:
- This needs to be changed if we don't accept false values.
- `None` should be used here instead of more complicated `${None}`.
- Could rephrase this like `Starting from Robot Framework 3.2, ...` to avoid the need for a separate note about versions.

Reply:
All are good suggestions, and I will handle them after we resolve the other issues